packages/core/src/payment/strategies/googlepay/googlepay.ts

```diff
@@ -30,6 +55,11 @@ export type GooglePayCreator = BraintreeModuleCreator<GooglePayBraintreeSDK>;
 
 export interface GooglePayPaymentOptions {
     environment: EnvironmentType;
+    paymentDataCallbacks?: {
+        onPaymentDataChanged?(
+            intermediatePaymentData: IntermediatePaymentData,
+        ): Promise<NewTransactionInfo | void>;
```

Comment:
Do we need void here?

Reply:
Yes we need void here. In our case :
`paymentDataCallbacks: {
                onPaymentDataChanged: async ({
                    callbackTrigger,
                }: IntermediatePaymentData): Promise<NewTransactionInfo | void> => {
                    if (callbackTrigger !== CallbackTriggerType.INITIALIZE) {
                        return;
                    }.....`